project_status/views/project.xml

```diff
@@ -0,0 +1,93 @@
+<odoo>
+  <!-- List View-->
+  <record id="view_project" model="ir.ui.view">
+    <field name="name">project.status.tree</field>
+    <field name="model">project.project</field>
+    <field name="inherit_id" ref="project.view_project"/>
+    <field name="type">tree</field>
+    <field name="field_parent">child_ids</field>
+    <field name="arch" type="xml">
+      <field name="user_id" position="after">
+        <field name="project_status"/>
+      </field>
+    </field>
+  </record>
+
+  <!-- Form View -->
+  <record id="edit_project" model="ir.ui.view">
+    <field name="name">project.status.form</field>
+    <field name="model">project.project</field>
+    <field name="inherit_id" ref="project.edit_project"/>
+    <field name="type">form</field>
+    <field name="arch" type="xml">
+      <data>
+        <xpath expr="//header" position="inside">
+          <field name="project_status"
+                 widget="statusbar"
+                 options="{'clickable': '1', 'fold_field': 'fold'}"/>
+        </xpath>
+      </data>
+    </field>
+  </record>
+
+  <!-- Simplified Form View -->
+  <record id="project_project_view_form_simplified" model="ir.ui.view">
+    <field name="name">project.status.simpleform</field>
+    <field name="model">project.project</field>
+    <field name="inherit_id" ref="project.project_project_view_form_simplified"/>
+    <field name="type">form</field>
+    <field name="arch" type="xml">
+      <data>
+        <field name="name" position="after">
+          <field name="project_status"
+                 widget="selection"

```

Comment:
@patrickrwilson IMHO here better to use `option={no_edit_create: true}` and remove `widget="selection"`